token-contracts/WorkerToken/src/main/java/finance/omm/score/WorkerTokenImpl.java

```diff
@@ -0,0 +1,159 @@
+package finance.omm.score;
+
+import static java.math.BigInteger.TEN;
+import static java.math.BigInteger.ZERO;
+
+import java.math.BigInteger;
+import java.util.List;
+
+import finance.omm.core.score.interfaces.WorkerToken;
+import finance.omm.utils.db.EnumerableSet;
+import score.Address;
+import score.ArrayDB;
+import score.Context;
+import score.DictDB;
+import score.VarDB;
+import score.annotation.EventLog;
+import score.annotation.External;
+import score.annotation.Optional;
+
+public class WorkerTokenImpl implements WorkerToken {
+
+    private static final String TAG = "Worker Token";
+
+    private static final String BALANCES = "balances";
+    private static final String TOTAL_SUPPLY = "total_supply";
+    private static final String DECIMALS = "decimals";
+    private static final String WALLETS = "wallets";
```

Comment:
The key "wallet" is same as that of python, in WorkerTokenImpl.java. However, the key is gets changed to "wallets_as_entries" in the EnumerableSetDB in the python version. So, this key should be changed to match the key of EnumerableSetDB in python, i.e. "wallets_as_entries".